### ISSUE TITLE
Joining forces?

### ISSUE THREAD

**Issue by lgalabru** [dmitrytorba/clarity#5]
Hi @dmitrytorba, this project look really promising!
How would you feel about solving the remaining blocking issues, and opening a PR in https://github.com/lgalabru/clarity-lsp? I'm happy to provide some support if you need help (I was unable to run this plugin on my machine).
At the root of the aforementioned repo, you'll find a directory `/editors`, it would be great to have an entry for `sublime` :).
Super excited by this project!!

**Comment by dmitrytorba** [comment]
Yeah, that sounds good to me! Thanks for offering support, it will be great to have you as a resource.  Also, good job on implementing the LSP server, its not a trivial undertaking.
